docs/competition/safeguarding_2020/safeguarding_2020.md

```diff
@@ -0,0 +1,41 @@
+
+This guidance is largely driven by the fact that we can no longer guarantee the involvement of the competitor's responsible adult in any interactions.
+
+## YouTube comments
+
+Most of the communication during the SR2020 competition is likely to be through comments on the livestream YouTube video.
+
+All normal advice regarding suitability of comments should be observed, in addition the following should be followed:
+
+- Personal contact details/social media accounts should not be shared or used.
+- No private discussions should be held between volunteers and competitors.
+- In the unlikely event that a competitor has to be contacted privately, another adult must be included in the discussion, ideally this should be both the competitor's responsible adult and another SR volunteer.
+- In the event that a competitor makes contact privately, these additional parties must be immediately included.
+
+## Competitor Interviews
+
+Some pre-recorded interviews with competitors are going to be displayed during the events.
+
+During recording of these interviews the following guidance should be observed:
+
+- Calls must involve 2 SR volunteers.
+- Calls must involve the competitor's responsible adult. This could be either their teacher joining the call, or their parent/carer.
+- Both SR volunteers must be in the call before the competitor joins. It is acceptable for this to be achieved merely by both volunteers joining the call at least 5 minutes before the competitor is expected to join.
+- Both SR volunteers and the responsible adult must leave the call after the competitor, excepting the circumstance whereby the call is ended for all parties simultaneously by the host.
+- It is acceptable for the responsible adult to be a parent merely in the room and not constantly visible to the camera. However, this would require the competitor to be on loud speaker (not using headphones), and the parent stating to the camera at the beginning that they will remain in the room throughout.
+- The intended responsible adult should be known in advance, however it is acceptable for it to change last minute.
+- Should the second SR volunteer or the responsible adult leave the call unexpectedly, e.g. through network dropout etc., then the call can continue uninterupted so long as the expectation is for the missing party to rejoin imminently. Should a few minutes pass and the missing party has not rejoined, or should it become apparent that they don't intend to rejoin, then the call should be ended and rescheduled.
+- The entire call should be recorded, and an unmodified original stored appropriately on SR systems.
+- Signed Media Consent Forms should be held by SR prior to the call beginning.
+
+
+## Incidents
+
+In the event of incident occuring that requires urgent guidance, contact details for Thomas Scarsbrook will be available.
+
+Less urgent incidents should just be reported through the standard SR safeguarding procedures. **Urgent events must also be reported**
```

Comment:
> **Urgent events must also be reported**

Is this just stating the obvious, or was there an additional point here? (This sentence feels incomplete somehow)

Reply:
Stating the obvious.

It needs to be clear that just because I've been phoned for something urgent, it doesn't cancel out the normal reporting process.

Reply:
Ah, that's more than I'd read into it. Perhaps:
```suggestion
All incidents should be reported through the standard SR safeguarding procedures, even if urgent guidance has been sought from Thomas Scarsbrook.
```

Reply:
This reads better for me